app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/TestRunner.ts

```diff
@@ -116,38 +123,63 @@ export class TestRunner {
     }
 
     this._testSuites.splice(index, 0, {
-      name,
+      name: applyMarkdown(name),
       buildSuite,
       testCases: [],
       nestingLevel: (this._currentTestSuite?.nestingLevel || 0) + 1,
-      only: !!(only || this._currentTestSuite?.only),
-      skip: !!(skip || this._currentTestSuite?.skip),
+      decorator: decorator ? decorator : this._currentTestSuite?.decorator ? this._currentTestSuite?.decorator : null,
```

Comment:
Could we avoid nesting inline conditions?

Reply:
Resolved in https://github.com/software-mansion/react-native-reanimated/pull/6047